tests/admin_docs/models.py

```diff
@@ -15,6 +15,12 @@ class Group(models.Model):
 
 
 class Family(models.Model):
+    """
+    This is first line with tag :tag:`extends`
+
+    This is a line with tag :tag:`extends`
+    """
```

Comment:
```suggestion
    """
    This is first line with tag :tag:`extends <built_in-extends>`

    This is a line with tag :tag:`extends`
    """
```
For supporting the `:tag:title <target> format`